libs/core/src/components/sage-tabs/sage-tabs.tsx

```diff
@@ -0,0 +1,164 @@
+import { Component, Element, Host, h, Prop, Listen } from '@stencil/core';
+@Component({
+  tag: 'sage-tabs',
+  styleUrl: 'sage-tabs.scss',
+  shadow: true,
+  // scoped: true,
+})
+export class SageTabs {
+  private tabs;
+  private tabPanels;
+  @Element() el: HTMLDivElement;
+
+  /**
+   * Sets the aria-label attached to the tablist element, required
+  */
+  @Prop() tablistLabel: string;
+ 
+  /**
+    * Sets unique id on tabs component, required
+  */
+  @Prop() componentId: string;
+  
+  /**
+   * Sets tabs variant styles as outlined in Figma documentation, optional
+   * @defaultValue primary
+    */
+  @Prop() variant?: 'primary' | 'availability' | 'filter' = 'primary';
+
+  /**
+   * Sets default active tab, required
+   */
+  @Prop({mutable: true}) activeTab: string;
+
+  @Listen('tabClick', {
+    passive: true,
+    target: 'body',
+  }) 
+  tabClickHandler(event: CustomEvent<string>) {
+    if (this.componentId === event.detail[1]) {
+      this.activeTab = event.detail[0];
+    }
+  }
+
+  private findAllChildren = () => {
+    this.tabs = this.el.querySelectorAll('sage-tab');
+    this.tabPanels = this.el.querySelectorAll('sage-tabpanel');
+  }
+
+  private passPropsToChildren = () => {
+    this.tabs.forEach(child => {
+      child['activeTab'] = this.activeTab.toString();
+      child['parentComponent'] = this.componentId.toString();
+      child['variant'] = this.variant.toString();
+    });
+    this.tabPanels.forEach(child => {
+      child['activeTab'] = this.activeTab.toString();
+      child['parentComponent'] = this.componentId.toString();
+      child['variant'] = this.variant.toString();
+    });
+  }
+
+  @Listen('keydown', {
+    passive: true,
```

Comment:
Explain more about the use of `passive` here

Reply:
I believe this is no longer needed, I was playing around with it when trying to scope `@Event` properly, removing now.